README.md

```
@@ -0,0 +1,172 @@
+
+<br>
+
+## 🐣 프로젝트 소개
+
+**성장형 취업 준비 서비스, Chick Mate**
+<br><br>
+Chick Mate의 대표 캐릭터 칰칰이와 함께 AI 면접관을 통해 면접을 연습하는 성장형 면접 준비 서비스입니다.
+면접을 준비하는데 어려움을 겪는 취업·이직 준비생이 각자에 맞는 역량을 키울 수 있도록 도와줍니다.
+
+> - **작업 기간** : 2025. 03. 20 ~ 2025. 04. 30
+> - **배포 주소** : www.chickmate.site
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)